test/client/spec/victory-shared-events/victory-shared-events.spec.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Client tests
+ */
+import { curry, flow, forEach } from "lodash";
+import { mount } from "enzyme";
+import React from "react";
+import { addEvents } from "src/index";
+import VictorySharedEvents from "src/victory-shared-events/victory-shared-events";
+import { MockVictoryComponent, MockDataComponent } from "../mock-components";
+
+describe("components/victory-shared-events", () => {
+  const EventedMockVictoryComponent = addEvents(MockVictoryComponent);
+
+  const findVictoryComponentByName = curry((name, component) => {
+    return component
+      .find(EventedMockVictoryComponent)
+      .filterWhere((victoryComponent) => {
+        return victoryComponent.props().name === name;
+      });
+  });
+
+  const findDataComponents = (component) => {
+    return component.find(MockDataComponent);
+  };
+
+  const findByIndex = curry((index, component) => {
+    return component.at(index);
+  });
+
+  const mapToEventEffects = curry((testFn, components) => {
+    return components.map(testFn);
+  });
+
+  const expectEqual = curry((expectation, testData) => {
+    expect(testData).to.eql(expectation);
+  });
+
+  const expectEventEffects = (expectationMatrix, testFn, component) => {
```

Comment:
Please re-write this for clarity. 

These methods might help: `.get()` and `.props()`.
http://airbnb.io/enzyme/docs/api/ReactWrapper/get.html
http://airbnb.io/enzyme/docs/api/ReactWrapper/props.html